hw-2/models/user.ts

```diff
@@ -0,0 +1,29 @@
+import * as Joi from '@hapi/joi';
+import { ContainerTypes, ValidatedRequestSchema } from 'express-joi-validation';
+
+/** User model. */
+export interface User {
+  id: string;
+  login: string;
+  password: string;
+  age: number;
+  isDeleted: boolean;
```

Comment:
We could make `isDeleted` to be optional: `isDeleted?: boolean`

Reply:
We even have to! Thanks!